x/incentives/keeper/grpc_query_test.go

```diff
@@ -0,0 +1,248 @@
+package keeper_test
+
+import (
+	"time"
+
+	"github.com/c-osmosis/osmosis/x/incentives/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (suite *KeeperTestSuite) TestGRPCPotByID() {
+	suite.SetupTest()
+
+	// create a pot
+	addr1 := sdk.AccAddress([]byte("addr1---------------"))
+	startTime := time.Now()
+	coins := sdk.Coins{sdk.NewInt64Coin("stake", 10)}
+	potID := suite.CreatePot(addr1, coins, types.DistrCondition{}, startTime, 2)
```

Comment:
what is the `2` parameter?

Reply:
the number of epoch this pot should exist after starting distribution. pot is paid in 2 epochs. (half/half)

Reply:
Can this be switched to a variable called `numEpochsPaidOver` and then that variable is passed here?

Reply:
It would be good